whereami/k8s/deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
               - all
           privileged: false
           readOnlyRootFilesystem: true
-        image: us-docker.pkg.dev/google-samples/containers/gke/whereami:v1.2.14
+        image: us-docker.pkg.dev/google-samples/containers/gke/whereami:v1.2.15
```

Comment:
In this file should you add the `8000` port for prometheus /metrics too?

Reply:
yes. can't use `8080` in gRPC mode when using Cloud Run because `8080` is used for the serving port.